swagger.yaml

```diff
@@ -3711,6 +3711,11 @@ paths:
           name: external_platform_name
           description: External platform name when platform type is set to external. The value of this parameter will be ignored if platform_type is not external.
           type: string
+        - in: query
+          name: high_availability_mode
+          type: string
+          enum: [ 'Full', 'None' ]
```

Comment:
would it make sense to set a default value? We would get rid of all the nil checks

Reply:
I was more questioning if we really need this parameter to be optional. AFAIK only the UI uses this endpoint, so I was hoping they would set it all the time since the high availability parameter is set at cluster creation, and cannot be updated afterwards.

Reply:
I also saw that when we create a cluster the default value is full, see: https://github.com/openshift/assisted-service/blob/master/swagger.yaml#L4758